CONTRIBUTING.md

```diff
@@ -18,8 +25,64 @@ A post-install action is thrown. This action first install dependencies with `le
 
 At this point you are ready to contribute.
 
+## Storybook
+
+This tool will help you display component in a page where it will be isolated (rather than integrated in a page with many components).
+
+We are using the state of the art tool: [Storybook](https://storybook.js.org/).
+
+To launch it, you can run:
+
+```sh
+npm run storybook
+```
+
+## Keeping your forked repository up-to-date
+
+Before you start developping, it is important to always keep your forked repository **up-to-date**.
+
+### Link your forked repository to the original project
+
+For this, you will add a remote which points to the original repository (this has to be done only once):
+
+```sh
+git remote add upstream https://github.com/AxaGuilDEv/react-toolkit
```

Comment:
Ce n'est pas plutôt `https://github.com/AxaGuilDEv/react-toolkit.git`

Reply:
The convention is:
- For bare repo: add the .git extension
- for repo with a working tree, don't...

So I will add it, no problem.

Reply:
Will fix, thanks